src/main/java/org/jenkinsci/plugins/badge/BadgeAction.java

```diff
@@ -5,36 +5,59 @@
 import jenkins.model.Jenkins;
 
 import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.QueryParameter;
-
 
 /**
 * @author Kohsuke Kawaguchi
 */
 public class BadgeAction implements Action {
+    /**
+     *
+     */
     private final BadgeActionFactory factory;
 
+    /**
+     *
+     */
     public final Job project;
-
+    /**
+     *
+     * @param factory
+     * @param project
+     */
     public BadgeAction(BadgeActionFactory factory, Job project) {
         this.factory = factory;
         this.project = project;
     }
-
+    /**
+     *
+     * @return
+     */
+    @Override
```

Comment:
[Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.